test/Python.C

```diff
@@ -227,14 +229,14 @@ def smapInto(r,f,x):
   if (x == None):
     return r
   else:
-    r.append(f(x[0]))
+    r.append(f(x[0]()))
     return smapInto(r,f,x[1])
 def smap(f,x):
   r=[]
   smapInto(r,f,x)
   return r
 
-if (sum(smap(lambda vs: sum(map(lambda v:v.x, vs)),f.stss)) != 207900):
+if (sum(smap(lambda vs: sum(map(lambda v:v.x, vs)),f.stss[1])) != 207900):
```

Comment:
Could you explain why this change had to be made?

Reply:
just fixed. now it is f.stss